statement.go

```diff
@@ -170,11 +172,14 @@ func (c converter) ConvertValue(v interface{}) (driver.Value, error) {
 	case reflect.Bool:
 		return rv.Bool(), nil
 	case reflect.Slice:
-		ek := rv.Type().Elem().Kind()
-		if ek == reflect.Uint8 {
+		switch t := rv.Type(); {
+		case t == jsonType:
+			return v, nil
+		case t.Elem().Kind() == reflect.Uint8:
 			return rv.Bytes(), nil
+		default:
+			return nil, fmt.Errorf("unsupported type %T, a slice of %s", v, t.Elem().Kind())
 		}
-		return nil, fmt.Errorf("unsupported type %T, a slice of %s", v, ek)
```

Comment:
Is this change needed?
It seems `t.Elem().Kind() == reflect.Uint8:` works fine for RawMessage.
https://play.golang.org/p/ADunkW0Za-9

Reply:
It’s needed, because you want the `driver.Value` to get as `json.RawMessage` to ` interpolateParams` and not as `[]byte` (#1058 won’t work without it).